manifests/0000_10_kube-apiserver-operator_06_deployment.yaml

```diff
@@ -43,6 +43,8 @@ spec:
           value: quay.io/openshift/origin-hypershift:v4.0
         - name: OPERATOR_IMAGE
           value: docker.io/openshift/origin-cluster-kube-apiserver-operator:v4.0
+        - name: OPERAND_IMAGE_VERSION
+          value: "0.0.1-snapshot_kubernetes"
```

Comment:
@sanchezl write us a test that makes sure this gets substituted properly and ends up as `1.XX.X`  do not hard code a number.

Reply:
Opened https://github.com/openshift/cluster-kube-apiserver-operator/pull/367